controller/wingsail/wingsail_ctrl_node.py

```diff
@@ -78,9 +80,29 @@ def __init_publishers(self):
         """Initializes the publishers of this node. Publishers update ROS topics so that other ROS
         nodes in the system can utilize the data produced by this node.
         """
-        # TODO Implement this function by initializing publishers for topics that give the desired
-        # output data
-        pass
+
+        self.get_logger().debug("Initializing publishers...")
+        self.__trim_tab_angle_pub = self.create_publisher(
+            msg_type=SailCmd,
+            topic="sail_cmd",
+            qos_profile=1,
+        )
+
+    def __init_timer_callbacks(self):
+        self.get_logger().debug("Initializing timer callbacks...")
+
+        # Publishing data to ROS topics
+        self.create_timer(
+            timer_period_sec=self.pub_period,
+            callback=self.__publish,
+        )
+
+    # PUBLISHER CALLBACKS
+    def __publish(self):
+        sailcmd = SailCmd()
+        sailcmd.trim_tab_angle_degrees = -15.5
```

Comment:
Change the variable name `sailcmd` to `msg` just to be consistent with the rest of the codebase. Also, consider changing `-15.5` back to `0.0`.